使用しているパッケージのバージョン問題で、npm installに失敗する

@types/reactのバージョンがReact18系のものになっており、npm installに失敗する模様でした。

@types/reactのバージョンを使われているライブラリとの兼ね合いで、
17系までダウングレードするとnpm installは成功するようでした。


エラーログを貼らせていただきます。

```
atobe_tatsuya@atobetatsuyanoiMac next-simple-template % npm i                    
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: @testing-library/react-hooks@8.0.0
npm ERR! Found: @types/react@18.0.9
npm ERR! node_modules/@types/react
npm ERR!   dev @types/react@"^18.0.9" from the root project
npm ERR!   peer @types/react@"*" from @design-systems/utils@2.12.0
npm ERR!   node_modules/@devtools-ds/themes/node_modules/@design-systems/utils
npm ERR!     @design-systems/utils@"2.12.0" from @devtools-ds/themes@1.2.0
npm ERR!     node_modules/@devtools-ds/themes
npm ERR!       @devtools-ds/themes@"^1.2.0" from @devtools-ds/object-inspector@1.2.0
npm ERR!       node_modules/@devtools-ds/object-inspector
npm ERR!         @devtools-ds/object-inspector@"^1.1.2" from @storybook/addon-interactions@6.5.9
npm ERR!         node_modules/@storybook/addon-interactions
npm ERR!       1 more (@devtools-ds/tree)
npm ERR!   1 more (@types/react-syntax-highlighter)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peerOptional @types/react@"^16.9.0 || ^17.0.0" from @testing-library/react-hooks@8.0.0
npm ERR! node_modules/@testing-library/react-hooks
npm ERR!   dev @testing-library/react-hooks@"^8.0.0" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: @types/react@17.0.48
npm ERR! node_modules/@types/react
npm ERR!   peerOptional @types/react@"^16.9.0 || ^17.0.0" from @testing-library/react-hooks@8.0.0
npm ERR!   node_modules/@testing-library/react-hooks
npm ERR!     dev @testing-library/react-hooks@"^8.0.0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /Users/atobe_tatsuya/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/atobe_tatsuya/.npm/_logs/2022-07-28T02_19_38_114Z-debug-0.log

```

--- a/package.json
+++ b/package.json
@@ -42,7 +42,7 @@
     "@testing-library/react": "12.1.5",
     "@testing-library/react-hooks": "^8.0.0",
     "@types/node": "^17.0.35",
-    "@types/react": "^18.0.9",
+    "@types/react": "^17.0.48",
     "@typescript-eslint/eslint-plugin": "5.20.0",
     "@typescript-eslint/parser": "5.20.0",
     "css-loader": "^6.7.1",

